Themis/Views/Assessment/FiletreeSidebar/FiletreeSidebarView.swift

```diff
@@ -14,6 +14,11 @@ struct FiletreeSidebarView: View {
                 List {
                     OutlineGroup(fileTree, id: \.path, children: \.children) { tree in
                         Text(tree.name)
+                            .bold(tree == cvm.selectedFile)
+                            .padding(.horizontal)
+                            .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .leading)
+                            .background(tree == cvm.selectedFile ? Color(UIColor.systemGray5) : Color(UIColor.systemBackground))
+                            .cornerRadius(10)
```

Comment:
I think it would be more efficient to use tree === cvm.selectedFile, so comparing the reference because
using the == Operator it will calculate the path for both sides recursively and comparing the arrays afterwards.